ingest/CreateArrayImportTsvs.wdl

```diff
@@ -63,6 +66,7 @@ task CreateImportTsvs {
         ~{"--probe-info-file " + probe_info_file} \
         ~{"--probe-info-table " + probe_info_table} \
         -SNM ~{sampleMap} \
+        ~{true="--use-compressed-data" false="" use_compressed_data} \
```

Comment:
i'm confused about what `false=""` does. 
why wouldn't this be:
~{"--use-compressed-data" use_compressed_data}  \

Reply:
I don't think that works in WDL because it's a boolean. But, because you're going to change the default in GATK I'm thinking it's better to make it explicit anyway (and it looks cleaner):

```--use-compressed-data ~{use_compressed_data}```